tsconfig.json

```diff
@@ -21,9 +25,20 @@
     ],
     "baseUrl": ".",
     "paths": {
-      "~/*": ["./src/*"]
+      "~/*": [
+        "./src/*"
+      ]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "decs.d.ts"],
-  "exclude": ["node_modules"]
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    ".next/types/**/*.ts",
+    "decs.d.ts",
+    "/Users/maiasasson/work/sum-it-up/.next/types/**/*.ts"
```

Comment:
need to switch to relative path

Reply:
and do we even need it btw.. because we already have this on line 37?

Reply:
Not sure, it does it every time I do `yarn lint` didn't realize I added it